test/web-platform-tests/to-upstream/html/browsers/browsing-the-web/navigating-across-documents/fragment-replace.html

```diff
@@ -11,50 +11,47 @@
 setup({ allow_uncaught_exception: false });
```

Comment:
OK, per my meandering in https://github.com/whatwg/html/pull/2869 I figured this out, so this is all good.